ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -1,5 +1,75 @@
 [
   {
+    "id": "java-mysql",
+    "creator": "ide",
+    "name": "Java-MySQL",
+    "description": "Multi-machine environment with Default Java Stack and MySQL database",
+    "scope": "general",
+    "tags": [
+      "Java 1.8, Tomcat 8, MySQL 5.7"
+    ],
+    "components": [
```

Comment:
@eivantsov - I think there is a difference between components and tags in this document. The tags are used for filtering and they are a different list of labels. But I believe the components are the list that appears in the UD.

If we have a long list of components, are we concerned about the display height of the stack boxes?  @ashumilova - if we have a super long list of components, do we just cut off the list or do we make the box taller?  We probably need to think about what to do here.

But I would think that we should go ahead and make the components list complete and then as a secondary measure work on improving the UX display of the stack box?